src/styles/box/card.scss

```diff
@@ -0,0 +1,168 @@
+.catalog {
+  display: grid;
+  grid-template-rows: 408px;
+  grid-template-columns: 200px;
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @media screen and (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media screen and (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media screen and (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(4, 1fr);
+    padding-left: 128px;
+    padding-right: 128px;
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: $card-width;
```

Comment:
The variable `$card-width` is used here but not defined. Ensure that this variable is defined in a file that is imported before this one, or define it directly here.